### ISSUE TITLE
A11y_AzureDataStudio_Runquery_ScreenReader: ORCA screen reader is not announcing "17:14:34" which appears under the 'Messages' tab

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22673]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**#GitHub Tags:**
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG1.3.1; #Linux; #Element:TabItem; #ScreenReader; #ORCA;

**Actual Result:​**
ORCA screen reader is not announcing "17:14:34" which appears under the 'Messages' tab.

**Expected Result:​**
ORCA screen reader should announce "17:14:34" which appears under the 'Messages' tab.

**Environment Details:​**
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

**Repro Steps:**

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to "File" and hit ENTER key. TAB to "New Query" and hit ENTER key.
4. TAB to "Run" button and hit ENTER key. Start screen reader.
5. TAB to "Messages" tab item and observe that ORCA screen reader is not announcing "17:14:34" which appears under the 'Messages' tab.

### User Impact:
Screen reader users will not get information about the "17:14:34" which appears under the 'Messages' tab.

### Attachment:
[Video 3-30 at 12.37 (5).webm](https://user-images.githubusercontent.com/90901519/228828239-beb2b941-aef1-43d9-a7b9-1eec6400c393.webm)

**Comment by PoojaNamde** [comment]
Primary bug: https://github.com/microsoft/azuredatastudio-mysql/issues/180

**Comment by alanrenmsft** [comment]
@PoojaNamde could you please try the latest insiders build, I've fixed the issue.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:06-06-23

Verified this issue in the latest env Version: 1.45.0-insider
Commit: d414c8de56a189b5f85c28ba18d9704ce62c7ba0
Date: 2023-06-05T05:46:00.131Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 5.19.0-43-generic.

Issue is fixed, hence closing the bug.
[Video 6-06 at 12.24.webm](https://github.com/microsoft/azuredatastudio/assets/90901519/8e7fa873-8a1a-4485-8e14-a61b80ae4d5a)
